### ISSUE TITLE
Suggestion: Audio Amplification

### ISSUE THREAD

**Issue by Defreezer** [argorar/WebMConverter#74]
An option to set audio loudness in the Processing section would be very welcome.
Due to the varying audio levels for different applications (some of them less finely-tuned by their creators than others), my recordings usually have outputs at low volume, as using them at higher volume would make them rather loud on my system. I always need to amplify audio tracks in a different software like Audacity, then replace the original recording's audio before processing a webm at a suitable, hearable volume. Other sources, like video/audio files downloaded from the web, also have low volume at times. Having an Amplify option would be very convenient.
https://trac.ffmpeg.org/wiki/AudioVolume

In short, a filter that could do as exemplified in this image to remove the need for further editing:

![457465](https://github.com/argorar/WebMConverter/assets/79490923/94fdbe4e-3307-4996-b07f-d2d72707ce66)

**Comment by argorar** [comment]
Tested filter shared and not a big difference without.

**Comment by argorar** [comment]
Check latest version.
